homeassistant/components/python_script/__init__.py

```diff
@@ -105,7 +105,10 @@ def discover_scripts(hass):
 
     def python_script_service_handler(call):
         """Handle python script service calls."""
-        execute_script(hass, call.service, call.data)
+        try:
+            execute_script(hass, call.service, call.data)
+        except ValueError:
```

Comment:
This cannot happen. This handler is registered based on the files that exist below.